app/components/CompanyValues.tsx

```diff
@@ -0,0 +1,150 @@
+// LIBRARY IMPORTS
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  Icon,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import Image from "next/image";
+import { FaRegClock, FaPeopleGroup, FaRegStar } from "react-icons/fa6";
+
+// LOCAL IMPORTS
+import gearBackground from "../../public/images/machine-gear-background.png";
+
+// VALUES COMPONENT DEFINITION
+export default function CompanyValues() {
+  return (
+    <>
+      <Image
+        src={gearBackground}
```

Comment:
```suggestion
        alt=""
        src={gearBackground}
```
As it's a decorative image, it's best to use an empty alt

Reply:
There is also warning in console _"Image with src "/_next/static/media/machine-gear-background.ffebf17d.png" has "fill" and parent element with invalid "position". Provided "static" should be one of absolute,fixed,relative."_